rpc/core/events.go

```diff
@@ -22,19 +22,20 @@ import (
 // string (escaped with single quotes), number, date or time.
 //
 // Examples:
-//		tm.event = 'NewBlock'								# new blocks
-//		tm.event = 'CompleteProposal'				# node got a complete proposal
+//		tm.event = 'NewBlock'				# new blocks
+//		tm.event = 'CompleteProposal'		# node got a complete proposal
 //		tm.event = 'Tx' AND tx.hash = 'XYZ' # single transaction
-//		tm.event = 'Tx' AND tx.height = 5		# all txs of the fifth block
-//		tx.height = 5												# all txs of the fifth block
+//		tm.event = 'Tx' AND tx.height = 5	# all txs of the fifth block
+//		tx.height = 5						# all txs of the fifth block
 //
 // Tendermint provides a few predefined keys: tm.event, tx.hash and tx.height.
-// Note for transactions, you can define additional keys by providing tags with
+// Note for transactions, you can define additional keys by providing events with
 // DeliverTx response.
 //
 //		DeliverTx{
-//			Tags: []*KVPair{
-//				"agent.name": "K",
+//			Events: []Event{
+//				{"agent": {{"name": "K"}},
```

Comment:
Don't we have a `type` field in the event now? Not clear how this maps to the ABCI struct.

Reply:
Updated.